Makefile

```diff
@@ -1,51 +1,15 @@
-VERSION ?= $(shell git describe --tags --abbrev=0 --exact-match 2>/dev/null)
```

Comment:
This file was moved as-is, to the `lint` directory. A new Makefile was added to a root, to test/check all submodules